tests/cypress/support/views.js

```diff
@@ -214,7 +214,7 @@ export const action_createPolicyFromSelection = (uPolicyName, create=true, polic
   if (!policyConfig['cluster_binding'] && Cypress.env('MANAGED_CLUSTER_NAME') !== undefined) {
     policyConfig['cluster_binding'] = [`name: "${Cypress.env('MANAGED_CLUSTER_NAME')}"`]
   }
-  if (policyConfig['cluster_binding'].length > 0) {
+  if (policyConfig['cluster_binding']?.length > 0) {
```

Comment:
What does this do?

Reply:
When I was running the tests locally, it kept failing here because `policyConfig['cluster_binding']` was undefined, so it couldn't get the length. This makes it give `null` instead of throwing an error (and as a weird javascript thing, `(null > 0) === false`).

If I had some other problem in my local setup that probably caused this, then we can revert it.

Reply:
I see. This is equal to `policyConfig['cluster_binding'] && policyConfig['cluster_binding'].length > 0`

Reply:
Yeah - and your version is more obvious, so I'll update that